k8s/namespaces/wa/wa-task-batch-ad-hoc-pending-termination-tasks/prod-00.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: wa-task-batch-ad-hoc-pending-termination
```

Comment:
Since this is for prod will this start running in prod as soon as we merge?  Or is there any additional change to wire in these files into a kustomization required?